src/maintenance/performance.rst

```diff
@@ -99,6 +99,7 @@ will be continuously replicating.
 Erlang
 ------
 
+[The documentation no longer speaks about maximum connections.  Not sure how to change this.]
```

Comment:
Did not know whether it was outdated or missing :D
> On Jun 1, 2016, at 2:36 PM, Alexander Shorin <notifications@github.com> wrote:
> 
> In src/maintenance/performance.rst <https://github.com/apache/couchdb-documentation/pull/47#discussion_r65437455>:
> 
> > @@ -99,6 +99,7 @@ will be continuously replicating.
> >  Erlang
> >  ------
> >  
> > +[The documentation no longer speaks about maximum connections.  Not sure how to change this.]
> I guess by removing outdated block? (:
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/apache/couchdb-documentation/pull/47/files/71d7e0551710a4264fc9b185336f5baa27cbb210#r65437455>, or mute the thread <https://github.com/notifications/unsubscribe/AAA7TR7zJt6GKdYAEhmbqEIt0t2EsR4zks5qHe1qgaJpZM4IBg-B>.
>